linux_os/guide/services/ntp/chronyd_no_chronyc_network/rule.yml

```diff
@@ -15,8 +15,6 @@ rationale: |-
 
 severity: unknown
 
-platform: machine  # The check uses service_... extended definition, which doesnt support offline mode
```

Comment:
Containers have chrony too (or could). Reverting this change.

Reply:
This is very unlikely, as you need privileges to alter the system time. Container's time is the host's time.